sound/soc/sof/intel/hda-loader.c

```diff
@@ -252,10 +252,9 @@ int hda_cl_trigger(struct device *dev, struct hdac_ext_stream *hext_stream, int
 		return hda_dsp_stream_trigger(sdev, hext_stream, cmd);
 	}
 }
-EXPORT_SYMBOL_NS(hda_cl_trigger, SND_SOC_SOF_INTEL_HDA_COMMON);
 
-int hda_cl_cleanup(struct device *dev, struct snd_dma_buffer *dmab,
-		   struct hdac_ext_stream *hext_stream)
+static int hda_cl_cleanup(struct device *dev, struct snd_dma_buffer *dmab,
+			  struct hdac_ext_stream *hext_stream)
```

Comment:
this was exported for a reason, it'll be needed in the BPT/BRA stuff, see https://github.com/thesofproject/linux/pull/4679/commits/af779a3ca9bae844fb1691caaba7243e49afbe71

If you remove the export, you'll have to add it back at some point.

Reply:
OK, dropping this patch